config/hardware/extruder/sherpa_mini.cfg

```
@@ -0,0 +1,28 @@
+[gcode_macro _USER_VARIABLES]
+variable_extruder_enabled: True
+gcode:
+
+
+[extruder]
+# Sherpa mini Gear Ratio
+# new_rd = previous_rd * mesured_distance / requested_distance
+rotation_distance: 22.4748405
```

Can you replace it with this one that is the "standard" one?

```suggestion
rotation_distance: 22.67895
```